docs/src/pages/components/slider/UnstyledSlider.tsx

```diff
@@ -47,15 +46,21 @@ const StyledSlider = styled(SliderUnstyled)(
     border-radius: 50%;
     outline: 0;
     border: 2px solid currentColor;
-    background-color: ${theme.palette.getContrastText(theme.palette.primary.main)};
+    background-color: white;
```

Comment:
So far, we have been using the hex color. I would propose the same. 

```suggestion
    background-color: #fff;
```